README.md

```diff
@@ -102,23 +107,27 @@ See [variables.tf] and [examples/] for details and use-cases.
 
 #### Main Resource Configuration
 
-- **`name`**: *(Optional `string`, Forces new resource)*
+- [**`name`**](#var-name): *(Optional `string`)*<a name="var-name"></a>
 
   The name of the role.
-  Invalid characters will be replaced with dashes.
-  If omitted, Terraform will assign a random, unique name.
+Invalid characters will be replaced with dashes.
+If omitted, Terraform will assign a random, unique name.
+Forces new resource.
 
-- **`name_prefix`**: *(Optional `string`, Forces new resource)*
+- [**`name_prefix`**](#var-name_prefix): *(Optional `string`)*<a name="var-name_prefix"></a>
 
   If omitted, Terraform will assign a random, unique name.
-  Invalid characters will be replaced with dashes.
-  Creates a unique name beginning with the specified prefix.
-  Conflicts with name.
+Invalid characters will be replaced with dashes.
+Creates a unique name beginning with the specified prefix.
+Conflicts with name.
+Forces new resource.
```

Comment:
@thiesen can we render this with indent? the result will be the same so no real issue.. but makes the markdown more readable..